### ISSUE TITLE
Build instructions need updating

### ISSUE THREAD

**Issue by timstableford** [aetilius/pHash#9]
The build instructions in INSTALL still reference automake rather than CMake

**Comment by starkdg** [comment]
Just created a new branch for v1.0.0.   The cmake build totally redone.

**Comment by mcmasjc3** [comment]
Indeed.  I am trying to compile the Java bindings and there are no instructions for doing that.  The README says ./configure --enable-java, but there is no configure script except in the bindings/php directory.

**Comment by obushnell** [comment]
A little late, but still better than never. Please review and approve my fix for this issue. Thank you.

**Comment by starkdg** [comment]
This project is no longer being maintained.   But here is a new version.  It is not just a fork, but a complete redo.  

https://github.com/starkdg/phash

**Comment by obushnell** [comment]
And yet, this project has newer commits then yours. How does that work?

**Comment by starkdg** [comment]
@bushnell ,  I think you'll admit those are fairly minor commits.  I just cleaned it up a bit and added some unit tests.   I would have added it here, but the owner doesn't want to work on it anymore.   Not trying to bad mouth the project here, just letting you know.

**Comment by obushnell** [comment]
I have added a link to the new repo along with a message that this repo is no longer maintained to my pull request. Please approve it and officially indicate your position.

**Comment by starkdg** [comment]
@obushnell  I apologize.  I thought you owned the repos. 

No need to include a link or officially declare it dead.  I thought I was just stating the obvious, but people can decide for themselves which one suits them better.   I was merely supplying a link to a version that I had cleaned up and added tests.   I worked with the owner on this off and on, on various projects going back 10 years, but he apparently does not have the time or inclination to do much with it, but also does not want to give up the account.   I have no idea if he is even around anymore.  Not that there is anything more to do with it, because there are better solutions now.    (For image perceptual hashes,  see the phashml project under my page for a more robust image hash).  

If you want to talk more, you are welcome to message me on telegram @starkdg

**Comment by aetilius** [comment]
Addressed in #57 (merged). The new `README.md` documents the CMake build:

```sh
mkdir build && cd build
cmake -DPHASH_EXAMPLES=ON -DWITH_AUDIO_HASH=ON -DWITH_VIDEO_HASH=ON ..
make
sudo make install
```

The old autotools-era `INSTALL` file is left in place as a historical reference. Closing.
